### ISSUE TITLE
Remove 36 plate limitation

### ISSUE THREAD

**Issue by laukejas** [SoftFever/OrcaSlicer#7252]
### Is there an existing issue for this feature request?

- [X] I have searched the existing issues

### Is your feature request related to a problem?

Currently, OrcaSlicer limits a project to 36 plates max, a limitation inherited from BambuSlicer. This can be a problem when estimating time and material consumption for very large projects, as well as printing them and keeping things organized, requiring inconvenient workflows, such as removing some parts from the plates and adding others in their place to work around this limitation. For example, I have a project with 150 parts, where each part needs to be printed separately, meaning 150 different plates, and it is very difficult to properly plan it out.

### Which printers will be beneficial to this feature?

All

### Describe the solution you'd like

It would be nice if there was no plate limitation, or the limit would be configurable by the user (based on their computer specs, etc.).

**Comment by laukejas** [comment]
Bumping the issue due to risk of it becoming stale.

**Comment by niklasb22** [comment]
@laukejas I [tested ](https://github.com/niklasb22/OrcaSlicer/commit/9983cf84cf9de60195d606c4d5c30921861d2620)increasing the max plate count to 255 and it works fine without technical issues.
That said, with that many plates the user experience gets pretty clunky.
I’d suggest bumping the limit to something like 64 plates instead — still a big improvement, but easier to manage in practice.

**Comment by laukejas** [comment]
@niklasb22 Thank you for testing this! Why not make it unlimited, or at least 255 though? I get it, it gets clunky, but it's up to the user to deal with it or no. I don't see any advantage to limiting it to 64 plates. After all, user will only notice the clunkiness if they manually add that many plates, which I suspect will be pretty rare, so it won't affect casual users.

**Comment by tina-junold** [comment]
I ran into this today while planning the T250. Hope this will be changed in future?

**Comment by binafa** [comment]
I also need more plates for a large project. Is there any way to change this limit via a config file or something? Or is this limit hardcoded?

**Comment by keybl** [comment]
I agree, I have a project where 70 models are printed and they need to be printed separately. I would also like to see how much time and material it will take. Give us the ability to configure the number of maximum tables in the program settings.

**Comment by NanashiTheNameless** [comment]
Just so people know, My fork currently supports 64 plates and I am experimenting with bumping it to 96 but I am unlikely to go higher than that for performance reasons.
<https://github.com/NanashiTheNameless/OrcaSlicer>

The experimental 96 plate build is running and can be found here:
<https://github.com/NanashiTheNameless/OrcaSlicer/actions/runs/21106158529>
(<https://nightly.link/NanashiTheNameless/OrcaSlicer/actions/runs/21106158529>)
